pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -199,6 +199,8 @@ func commonArgs(cluster string, cfg *config.Cluster, networkName string, nodeNam
 		args = append(args, "-e", fmt.Sprintf("%s=%s", key, val))
 	}
 
+	args = append(args, "-e", "CGROUP_ROOT=/kubelet")
```

Comment:
podman does not need it?

Reply:
True, I probably do need to ensure podman support is still working.

Reply:
I'm trying to get this working with podman, but I can't get the podman provider working at all on my machine, unfortunately: https://github.com/kubernetes-sigs/kind/issues/1757 any ideas how I can get past that error?

Reply:
Then just add a comment in the podman file with a TODO explaining what is needed, don't worry about it. 
Thanks for the effort

Reply:
Okay cool, I have made it internally configurable so that a cgroup root is not passed through to podman and added a TODO comment.